deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,73 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+/* Algemeen */
+ul {
+    display: flex;
+    font-family: 'Poppins', sans-serif;
+    opacity: 0.7;
+}
+
+/* Opdracht 1 */
+
+.liNoWrap1 {
+    flex-wrap: nowrap;
+    justify-content: space-between;
+}
+
+/* Opdracht 2 */
+
+.liContainers {
+    justify-content: space-evenly;
+}
+
+.liContainers li {
+    flex-basis: 33.3%;
+}
+
+/* Opdracht 3 */
+
+.container3 {
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.first-li {
+    align-self: flex-start;
+}
+.second-li {
+    align-self: center;
+}
+.third-li {
+    align-self: flex-end;
+}
+
+/* Opdracht 4 */
+
+.container4 {
+    flex-wrap: wrap;
+    gap: 10px 10px;
+}
+
+/* Opdracht 5 */
+.container5 {
+    display: flex;
+    justify-content: space-between;
+}
+
+.ul1 {
```

Comment:
Alles wat je hier op specifiek ul(cijfer) niveau herhaalt, kan je ook samenvoegen en alleen de specifieke dingen op ul(cijfer) niveau specificeren. Ik bedoel dan, in dit specifieke voorbeeld, dit: 
ul {
flex-direction: column;
gap: 40px;
} 

.ul1 {
 align-items: flex-start;
}

.ul2 {
 align-items: center;
}
 
.ul3 {
 align-items: flex-end;
}